apps/web/src/components/fancy/variable-font-cursor-proximity.tsx

```
@@ -38,7 +38,7 @@ const VariableFontCursorProximity = forwardRef<HTMLSpanElement, TextProps>(
           .map((s) => s.trim())
           .map((s) => {
             const [name, value] = s.split(" ");
-            return [name.replace(/['"]/g, ""), parseFloat(value)];
+            return [name?.replace(/['"]/g, ""), parseFloat(value ?? "0")];
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)